src/lib/helpers.js

```diff
@@ -0,0 +1,21 @@
+export const mustShowSelectionBar = state => state.ui.showSelectionBar || state.ui.selected.length !== 0
```

Comment:
The only part of the PR I'm not too sure about is this one.

I agree that these functions are not reducers; however, they do compute things based on the `state`, so I think they qualify as selectors. As such, they are closer to reducers than to independent helper functions.
Files has similar code and it's still with the reducers. So yeah, not too sure this is a good move.

Can we get a third opinion? @goldoraf or @gregorylegarec maybe?